lib/bolt/target.rb

```diff
@@ -65,7 +65,7 @@ def safe_name
     end
 
     def target_alias
-      inventory_target.target_alias
```

Comment:
I thought this was added in the inital GH-1200 implementation, but i guess not. I dont see a method or an instance variable access for `target_alias` on Inventory::Target. So what is it used for?

Reply:
@nicklewis Do you think it is OK to write this method like this? https://github.com/puppetlabs/bolt/pull/1340/files#diff-f0928cc9831d370263135e3cdb532fcbR68 It almost looks like maybe we just forgot a attr reader on Inventory::Target? It does not seem to be as simple as just exposing the @target_alias instance variable because it is currently only returning an empty array...

Reply:
Yes, we need to fix target_alias in general.